src/Raven.Server/Web/System/BackupServerWideHandler.cs

```diff
@@ -0,0 +1,230 @@
+            using (context.OpenReadTransaction())
+            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                var backups = ServerStore.Cluster.GetServerWideBackupConfigurations(context, taskName);
+                ServerWideBackupConfigurationResults backupsResult = new ServerWideBackupConfigurationResults();
+                
+                foreach (var backupBlittable in backups)
+                {
+                    var backup = JsonDeserializationServer.ServerWideBackupConfiguration(backupBlittable);
+                    backup.BackupDestinations = backup.GetDestinations();
+                    backupsResult.Results.Add(backup);
+                }
+                
+                context.Write(writer, backupsResult.ToJson());
+                writer.Flush();
+
+                return Task.CompletedTask;
+            }
+        }
+        
+        // Toggle Disable/Enable for specified task
+        [RavenAction("/admin/configuration/serve-wide/backup/state", "POST", AuthorizationStatus.ClusterAdmin)]
+        public async Task ToggleTaskState()
+        {
+            var disable = GetBoolValueQueryString("disable") ?? true;
+            var taskName = GetStringQueryString("taskName", required: false);
+           
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                // Get existing task
+                var backups = ServerStore.Cluster.GetServerWideBackupConfigurations(context, taskName); 
+                ServerWideBackupConfigurationResults backupsResult = new ServerWideBackupConfigurationResults();
+                
+                if (backups.Count() == 0)
+                    throw new InvalidOperationException($"Server-Wide Backup Task: {taskName} was not found in the server.");
+                
+                var serveWideBackupBlittable = backups.ToList()[0];
+                ServerWideBackupConfiguration serverWideBackup = JsonDeserializationServer.ServerWideBackupConfiguration(serveWideBackupBlittable);
+                
+                // Toggle
+                serverWideBackup.Disabled = disable; 
+            
+                // Save task
+                var (newIndex, _) = await ServerStore.PutServerWideBackupConfigurationAsync(serverWideBackup, GetRaftRequestIdFromQuery());
+                await ServerStore.WaitForCommitIndexChange(RachisConsensus.CommitIndexModification.GreaterOrEqual, newIndex);
+
+                using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+                {
+                    var test = new ModifyServerWideBackupResult() 
+                    {
+                        TaskName = taskName,
+                        RaftCommandIndex = newIndex
+                    };
+
+                    HttpContext.Response.StatusCode = (int)HttpStatusCode.OK;
```

Comment:
OK is the default, you do not need to set this.

Reply:
OK removed